pkgs/development/python-modules/pgsanity/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, postgresql }:
+
+buildPythonPackage rec {
+  pname = "pgsanity";
+  version = "0.2.9";
+  rev = "3bd391be1c5f0e2ce041652bdaf1d6b54424c6d8";
+
+  src = fetchFromGitHub {
```

Comment:
You may want to use `fetchPypi` here. If there's a certain reason why you decided to use the GitHub tarball, please leave a comment to avoid that anyone in the future hits that issue.

Reply:
@Ma27 I did want to use `fetchPypi` at first, but saw that the [last release date](https://pypi.org/project/pgsanity/#history) there was Apr 24 2017 and there had been [several minor commits](https://github.com/markdrago/pgsanity/commits/master) since then on `GitHub`.

However, these commits are mostly documentation and don't seem to affect any core functionality. If it is more appropriate, please let me know and I can quickly make the change to `fetchPypi`.

Reply:
The unreleased commits were mainly focused on getting a Docker setup working right (and one commit about FreeBSD in the README)? Correct me if I'm wrong, but I don't think that it's worth using the GitHub tarball rather than the PyPI tarball because of those commits.

(A valid reason would be missing tests in the PyPI tarball for instance)